sdcm/cluster_k8s/__init__.py

```diff
@@ -822,6 +843,10 @@ def deploy_scylla_cluster(self, node_pool: CloudK8sNodePool = None, node_prepare
                 - COMMON_CONTAINERS_RESOURCES['cpu']
                 - SCYLLA_MANAGER_AGENT_RESOURCES['cpu']
             )
+            if self.params.get('k8s_enable_performance_tuning') and self.CREATE_NODE_CONFIG_CR:
+                # NOTE: we should use at max 7 cores from 8.
+                if cpu_limit > 7:
+                    cpu_limit = math.ceil(cpu_limit / 8) * 7
```

Comment:
Can't get logic behind this, comment said that we should use at max 7 cores from 8, but what we should do if we have got more than 8, or less ? Current logic will give you 14 if `cpu_limit`=9

If you have any issue where it is discussed, could you please add it to the comment

Reply:
I will rephrase it. The idea is to use max 7 per each 8 cores. So if we have 32, we must use at max 28, not 30 or 31 using existing master logic. About 1 core from 8 is needed for IRQ processing.

Reply:
```suggestion
                    cpu_limit = cpu_limit - cpu_limit // 8
```

Reply:
Existing formula is more correct. Yours one will give wrong results when the diff is not 1 core.
For example, if we have 32 cores and "cpu_limit" gets calculated to 30 (it is really possible). Then your formula will return "27", but we expect it to be "28" which will be returned by existing formula.
So, I will leave it as is.

Reply:
Current formula will return `14` for `cpu_limit`=`11`, which will block pod from been deployed
I'am not advicating for any particular formula, but current is not correct, we need to fix it.

Reply:
done